packages/manager/modules/vps/src/dashboard/migrate/migrate.html

```diff
@@ -0,0 +1,113 @@
+<oui-tile-definition
+    data-term="{{:: 'vps_dashboard_migrate_title'| translate}}"
+>
+    <oui-tile-description>
+        <oui-select-picker
+            class="w-25 mt-3"
+            label="{{:: 'vps_dashboard_migrate_current_range_title' | translate}}"
+            disabled
+        >
+            <oui-select-picker-section>
+                <ul class="list-unstyled">
+                    <li>
+                        <span
+                            data-translate="vps_dashboard_migrate_configuration_label_core"
+                        ></span>
+                        <span
+                            data-translate="vps_dashboard_migrate_configuration_vcore"
+                            data-translate-values="{ count: $ctrl.stateVps.vcore }"
+                        ></span>
+                    </li>
+                    <li>
+                        <span
+                            data-translate="vps_dashboard_migrate_configuration_label_ram"
+                        ></span>
+                        <span
+                            data-ng-bind=":: $ctrl.getMemory($ctrl.stateVps.model.memory) | cucBytes:0:false:'GB'"
+                        ></span>
+                    </li>
+                    <li>
+                        <span
+                            data-translate="vps_dashboard_migrate_configuration_label_disk"
+                        ></span>
+                        <span
+                            data-ng-bind=":: $ctrl.stateVps.model.disk | cucBytes:0:false:'GB'"
+                        ></span>
+                    </li>
+                </ul>
+            </oui-select-picker-section>
+        </oui-select-picker>
+        <p class="mt-3" data-translate="vps_dashboard_migrate_information"></p>
+        <p>
+            <span data-translate="vps_dashboard_migrate_faq_title"></span>
+            <a
+                data-ng-href="{{:: $ctrl.MIGRATE_FAQ_LINK }}"
+                target="_blank"
+                rel="noopener"
+            >
+                <span
+                    data-translate="vps_dashboard_migrate_faq_link_text"
+                ></span>
+                <span
+                    class="oui-icon oui-icon-external-link"
+                    aria-hidden="true"
+                ></span>
+            </a>
+        </p>
+        <oui-select-picker
```

Comment:
same as above

Reply:
I am retaining `<select-picker>` here and added a `model` to bind. Since there is only one element it is always selected. And i can avoid writing class and customising styles